### PR TITLE
Merge common alias branch in final release branch 14.0.0

### PR DESCRIPTION
**Pull request by tanmaymanolkar1**
merge alias release branch into the final release branch.

Contains a breaking change which was marked as PATCH, updated the changelog entry to MAJOR and incremented the common version to reflect the same.
Wire new Broker Discovery Client into MSAL - still disabled by default (#2057)

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ V.NEXT
 - [MINOR] Fix issue for MSA only where headers are not propagated on web requests with different domain redirects on newer versions of WebView (88+) (#2072)
 - [PATCH] Add method to return list of broker that supports account manager (#2073)
 - [MINOR] Send client id as part of request bundle for getSsoToken Api (#2064)
-- [PATCH] Wire new Broker Discovery Client into MSAL - still disabled by default (#2057)
+- [MAJOR] Wire new Broker Discovery Client into MSAL - still disabled by default (#2057)
```
Comment (somalaya) [inline changelog.txt:11]: is this change intentional? Changing PATCH to MAJOR?

Comment (tanmaymanolkar1) [reply changelog.txt:11]: yes this is intentional.

**Review by somalaya — APPROVED** [review]
